pandas/core/indexing.py

```diff
@@ -1413,7 +1413,8 @@ def _get_slice_axis(self, slice_obj, axis=None):
 
 
 class _LocIndexer(_LocationIndexer):
-    """Purely label-location based indexer for selection by label.
+    """
+    Selects a group of rows and columns by label(s) or a boolean array.
```

Comment:
"Select" is not fully correct as it is not only for getting, but also for setting? 
Or is that general enough? (@jreback @TomAugspurger ) To set you of course also need to select the location to set .. 
The "see also" is using "access" now instead of "select"

Reply:
Ah yes, I had meant to switch to using `access` all around, although I'm not sure if that gets around the problem you mentioned. Perhaps, it's enough to mention you can use `loc` to get and set in the extended summary? I'm struggling to think of a good word that implies both getting and setting...but I will keep thinking on it.

Reply:
And I can also add some examples of using `loc` for setting values below.

Reply:
yes, that would be good